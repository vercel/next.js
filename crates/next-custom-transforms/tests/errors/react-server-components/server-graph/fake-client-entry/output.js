export default function() {
    return null;
}
