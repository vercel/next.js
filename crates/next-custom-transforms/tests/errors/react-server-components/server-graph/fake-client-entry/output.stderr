  x The "use client" directive must be placed before other expressions. Move it to the top of the file to resolve this issue.
   ,-[input.js:6:1]
 5 | // prettier-ignore
 6 | 'use client'
   : ^^^^^^^^^^^^
   `----
